src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -198,37 +198,55 @@ class HeaderHelpMenuUI extends Component<Props, State> {
       intl,
       opensearchDashboardsVersion,
       useDefaultContent,
-      opensearchDashboardsDocLink,
       surveyLink,
+      darkmode,
     } = this.props;
-    const { helpExtension, helpSupportUrl } = this.state;
+    const { helpExtension } = this.state;
 
     const defaultContent = useDefaultContent ? (
       <Fragment>
-        <EuiButtonEmpty href={opensearchDashboardsDocLink} target="_blank" size="xs" flush="left">
+        <EuiButtonEmpty
+          href="https://documentation.wazuh.com/4.9/"
```

Comment:
Perhaps the version could be dynamic to make it easier to maintain.

Reply:
Thanks, added on 3a26d552c763119ddde0346a5a9aa77150666838